Business Insider.py

```diff
@@ -0,0 +1,40 @@
+# importy
+from bs4 import BeautifulSoup
+import requests
+lst = list(range(1,30+1))
+section = 'https://businessinsider.com.pl/twoje-pieniadze'
+# wybieramy sekcje z której bierzemy artykuł
+for v in lst:
+    # Tworzymy listę linków z całej sekcji
+    page = section+'?page='+str(v)
+    result = requests.get(page)
+    soup = BeautifulSoup(result.text , "html.parser")
+    # zbieramy wszystkie darmowe artykuły ze strony
+    links = []
+    for link in soup.findAll('a'):
+        if section in link.get('href') and "subskrypcja?" not in link.get('href') :
+            links.append(link.get('href'))
+    for i in links:
+        # !!! pojedynczy artykuł do pliku
+        url = i
+        result = requests.get(url)
```

Comment:
Tu mógłbyś jeszcze sprawdzić, czy w odpowiedzi dostajesz status 200, w przeciwnym razie coś poszło nie tak (404 / 500 / jakiś timeout).

Mógłbyś obsłużyć inne statusy i np. przejść do kolejnego urla. Tak to w połowie skryptu może Ci się coś zepsuć i już kolejnych artykułów nie pobierzesz.